### PR TITLE
Initial Release of the Lnk.Bio certified connector

### PR DESCRIPTION
**Pull request by andreaolivato**
Lnk.Bio Certified Connector

---

- [x ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by andreaolivato** [comment]
@microsoft-github-policy-service agree [company="Lnk.Bio"]

**Comment by andreaolivato** [comment]
@microsoft-github-policy-service agree company="Lnk.Bio"

**Comment by andreaolivato** [comment]
@microsoft-github-policy-service agree company="Lnk.Bio"

File: certified-connectors/LnkBio/Readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Lnk.Bio Connector
+This is a connector for Lnk.Bio. Lnk.Bio is a link-in-bio and website builder web tool and mobile app favoured by 600,000 creators.
```
Comment (Amjed-Ayoub) [inline certified-connectors/LnkBio/Readme.md:2]: @andreaolivato all files should be free of grammatical errors and typos. Please change "favoured" to "favored".

File: certified-connectors/LnkBio/Readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Lnk.Bio Connector
```
Comment (Amjed-Ayoub) [inline certified-connectors/LnkBio/Readme.md:1]: @andreaolivato Please remove the word "Connector" from the title.

File: certified-connectors/LnkBio/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,141 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Automate publishing workflow on [Lnk.Bio](https://lnk.bio)",
```
Comment (Amjed-Ayoub) [inline certified-connectors/LnkBio/apiDefinition.swagger.json:4]: @andreaolivato please define a proper description for your connector. You can include any external links in your Intro.md file later in your ISV Studio submission. Please follow our documents [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)

File: certified-connectors/LnkBio/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    "title": "Lnk.Bio",
+    "version": "0.0.1",
```
Comment (Amjed-Ayoub) [inline certified-connectors/LnkBio/apiDefinition.swagger.json:6]: @andreaolivato since this is a new connector please change the version to "1.0"

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by andreaolivato** [comment]
All required corrections have been commited.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @andreaolivato,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**